codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -936,29 +940,30 @@ private String generateHttpBindingsValue(
             case SET:
                 // handle set as target shape
                 Shape targetValueSetShape = model.expectShape(targetShape.asSetShape().get().getMember().getTarget());
-                return getCollectionDeserializer(writer, model, targetValueSetShape, binding, operand);
+                return getCollectionDeserializer(writer, model, symbolProvider, targetValueSetShape, binding, operand);
             case LIST:
                 // handle list as target shape
                 Shape targetValueListShape = model.expectShape(targetShape.asListShape().get().getMember().getTarget());
-                return getCollectionDeserializer(writer, model, targetValueListShape, binding, operand);
+                return getCollectionDeserializer(writer, model, symbolProvider, targetValueListShape, binding, operand);
             default:
                 throw new CodegenException("unexpected shape type " + targetShape.getType());
         }
     }
 
     private String getCollectionDeserializer(
-            GoWriter writer, Model model,
+            GoWriter writer, Model model, SymbolProvider symbolProvider,
             Shape targetShape, HttpBinding binding, String operand
```

Comment:
nit: Style says that for function parameter definitions that span lines should be formatted such that each parameter is on its own line.